src/CoreBundle/MaintenanceMode/MaintenanceModeService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\MaintenanceMode;
```

Comment:
I intended to use the existing Maintenance namespace. Could you move again (maybe ChameleonSystem\CoreBundle\Maintenance\MaintenanceMode)?

Reply:
ok, changed

It's now stuttering: Maintenance\MaintenanceMode\MaintenanceModeService